train.py

```diff
@@ -24,102 +24,112 @@
 random.seed(seed)
 torch.cuda.manual_seed_all(seed)
 
+root = str(Path(__file__).parent.resolve())
+
+if "izvonkov" in root:
+    preference = "ivan"
+elif "hkjoo" in root:
+    preference = "kevin"
+
 # ----------------------------------------------------------------------------------------------------------------------
 # Parameters
 # ----------------------------------------------------------------------------------------------------------------------
 DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
 
 arg_parser = ArgumentParser()
-arg_parser.add_argument(
-    "--competition", type=str, default="south_africa", help="germany, south_africa"
-)
-arg_parser.add_argument("--model_type", type=str, default="spatiotemporal")
+
+# hyperparam
 arg_parser.add_argument("--batch_size", type=int, default=64)
 arg_parser.add_argument("--num_epochs", type=int, default=100)
+arg_parser.add_argument("--lr", type=float, default=0.001 if preference == "ivan" else 0.00001)
```

Comment:
Since you are using germany dataset often from now on, make sure to use 0.00001 for baseline, not 0.001 for LR